setup.cfg

```diff
@@ -0,0 +1,16 @@
+[flake8]
+# Line length sorted by black
+max-line-length = 1000
```

Comment:
Do we need this at all if its sorted by black? And does it have to be an insane number? :p